pages/experiences/list-storage-deals.js

```diff
@@ -0,0 +1,271 @@
+import * as React from "react";
+import * as System from "~/components/system";
+import * as Constants from "~/common/constants";
+
+import Group from "~/components/system/Group";
+import SystemPage from "~/components/system/SystemPage";
+import ViewSourceLink from "~/components/system/ViewSourceLink";
+
+import { createPow, ffsOptions } from "@textile/powergate-client";
+
+const PowerGate = createPow({ host: "http://pow.slate.textile.io:6002" });
+
+const EXAMPLE_CODE = `import * as React from 'react';
+import { FilecoinStorageDealsList, FilecoinRetrievalDealsList } from 'slate-react-system';
+import { createPow, ffsOptions } from "@textile/powergate-client";
+
+const PowerGate = createPow({ host: "http://pow.slate.textile.io:6002" });
+
+class Example extends React.Component {
+  componentDidMount = async () => {
+    const FFS = await PowerGate.ffs.create();
+    const token = FFS.token ? FFS.token : null;
+    PowerGate.setToken(token);
+    this.storageList = await PowerGate.ffs.listStorageDealRecords(
+      ffsOptions.withIncludeFinal(true),
+      ffsOptions.withIncludePending(true),
+      ffsOptions.withFromAddresses(
+        "t3ual5q5qo5wolfxsui4ciujfucqwf6gqso4lettcjwl2tyismgol7c4tngvoono5rmytuqotye7oosfjv6g7a",
```

Comment:
Lets pull these large values out into const values outside the closure and make the example feel a little more put together.